src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComInvokeAction.cs

```diff
@@ -15,11 +16,14 @@ namespace Microsoft.CSharp.RuntimeBinder.ComInterop
     /// </summary>
     internal sealed class ComInvokeAction : InvokeBinder
     {
+        [RequiresUnreferencedCode(Binder.TrimmerWarning)]
         internal ComInvokeAction(CallInfo callInfo)
             : base(callInfo)
         {
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
```

Comment:
```suggestion
        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

This is repeated everywhere. `IL2026` means the method is calling a method marked with `RequiresUnreferencedCode`. So using that name together helps people know what warning is being suppressed. (I know some have snuck in to existing code, I mean to update those to match everywhere.)

Reply:
I wasn't sure what that is for but I've seen we've used IL2026 with that text in other places so I followed the pattern. Will correct.